shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -898,6 +906,28 @@ void onUserLeaveHint() {
     }
   }
 
+  /**
+   * Invoke this from {@code Activity#onWindowFocusChanged()}.
+   *
+   * <p>A {@code Fragment} host must have its containing {@code Activity} forward this call so that
+   * the {@code Fragment} can then invoke this method.
+   */
+  void onWindowFocusChanged(boolean hasFocus) {
+    ensureAlive();
+    Log.v(TAG, "Received onWindowFocusChanged: " + (hasFocus ? "true" : "false"));
```

Comment:
This was probably useful for debugging but I think the flutter style is to not log unless something is actionable.

Reply:
Ok.  I was following the pattern in most of the other "on..." functions in this file to send a verbose log entry when they are called.  Should we remove the other log messages too?

Reply:
Feel free to leave it alone if the log message was intentional.